### PR TITLE
Enforce integral type for label images

### PR DESCRIPTION
**Pull request by ahojnnes**
Attempt to fix the 2.6 Travis failures.

### CODE REVIEW

**Comment by blink1073** [comment]
:fingers crossed:

File: skimage/measure/_regionprops.py [hunk 1]
```diff
@@ -491,6 +491,9 @@ def regionprops(label_image, intensity_image=None, cache=True):
     if label_image.ndim != 2:
         raise TypeError('Only 2-D images supported.')
 
+    if np.issubdtype(label_image.dtype, np.integer):
```
Comment (emmanuelle) [inline skimage/measure/_regionprops.py]: ``if not``? not sure I understand this test

Comment (ahojnnes) [inline skimage/measure/_regionprops.py]: Haha, yes sure. Just wrote the test on my machine without testing.

**Comment by blink1073** [comment]
Woohoo!  Fixes #1692.

**Pull request merged**